plugins/home/api-report.md

```diff
@@ -97,6 +101,15 @@ export type CustomHomepageGridProps = {
   preventCollision?: boolean;
 };
 
+// @public
+export const getVisitName: ({
+  rootPath,
+  document,
+}?: {
+  rootPath?: string | undefined;
```

Comment:
why `rootPath? : string | undefined` and not just `rootPath? : string`? Same q below for document.

Reply:
If you don't give `getVisitName` a rootPath or Document it will be filled with default values.

Reply:
Does this need to be exported at all?

Reply:
Thank you all, @soapraj asked me to update the api-reports on behalf of Renan (he is on vacation).
It seems that it is only used internally, so I updated the function to `@internal` and removed the export.
Here is the commit 🙌🏻 : https://github.com/backstage/backstage/pull/20020/commits/d65cb238e8a661c05a11f9c66740e419a8d61384